packages/xo-web/src/common/select-objects.js

```diff
@@ -859,8 +859,8 @@ export class SelectResourceSetsNetwork extends React.PureComponent {
 
   _getNetworks = createSelector(
     () => this.props.resourceSet,
-    ({ objectsByType }) => {
-      const { predicate } = this.props
+    () => this.props.predicate,
+    ({ objectsByType }, predicate) => {
```

Comment:
First input selector should be `() => this.props.resourceSet.objectsByType.network`.